src/controller/scan/options.go

```diff
@@ -44,3 +55,11 @@ func WithTag(tag string) Option {
 		return nil
 	}
 }
+
+// WithScanType set the scanType
+func WithScanType(scanType string) Option {
+	return func(options *Options) error {
```

Comment:
do we need to validate the param `scanType` before returning an `Option`?

Reply:
const value